### ISSUE TITLE
[Bug] Use default arguments instead of short circuiting or conditionals

### ISSUE THREAD

**Issue by peter-gribanov** [jupeter/clean-code-php#14]
You broke your code.
https://github.com/jupeter/clean-code-php#use-default-arguments-instead-of-short-circuiting-or-conditionals

Original is good

```php
function createMicrobrewery($name = null) {
    $breweryName = $name ?: 'Hipster Brew Co.';
    var_dump($breweryName);
}
createMicrobrewery();
createMicrobrewery('');
createMicrobrewery(null);
createMicrobrewery('foo');
```

Result

```
string(16) "Hipster Brew Co."
string(16) "Hipster Brew Co."
string(16) "Hipster Brew Co."
string(3) "foo"
```

Your "good"

```php
function createMicrobrewery($breweryName = 'Hipster Brew Co.') {
    var_dump($breweryName);
}
createMicrobrewery();
createMicrobrewery('');
createMicrobrewery(null);
createMicrobrewery('foo');
```

Result

```
string(16) "Hipster Brew Co."
string(0) ""
NULL
string(3) "foo"
```

Result is different. You broke your code.

**Comment by piotrplenik** [comment]
Good point. 
Do you have propose of better example?

**Comment by peter-gribanov** [comment]
Problem is fixed in #26
